drivers/input/Kconfig.stmpe811

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config INPUT_STMPE811
+	bool "STMPE811 touch driver"
+	default y
+	depends on DT_HAS_ST_STMPE811_ENABLED
+	select I2C
+	help
+	  Enable driver for STMPE811 touch panel.
+
+if INPUT_STMPE811
```

Comment:
Just use `depends on INPUT_STMPE811` in the option below, that's the common way of doing it if you only have one option in the block

Reply:
Once the thread was deleted, it was no longer needed